docs/_includes/DeveloperGuide/Requirements/02_use_cases.md

```diff
@@ -71,7 +71,17 @@ Actor: New User
 
 ---
 
-**UC05 - Mark and Unmark Expenses**
+**UC05 - List all Person**
+
+**MSS**
+1. A user requests to list all person after searching for a specific person.
+2. The system resets the displayed filtered list.
+3. The system displays the list of all person.
+   <br>Use case ends.<br>
+
+---
+
+**UC06 - Mark and Unmark Expenses**
 
 **MSS**
 1. A user requests to list expenses
```

Comment:
<img width="740" alt="Screenshot 2024-11-10 at 6 11 52 PM" src="https://github.com/user-attachments/assets/0be4378f-4142-416b-8e3a-6b70fc07846a">

From the issue, I think want us to use the above syntax so it will be something like:

1. A user requests to _list expenses (UC10)_

<Skips the `The system shows a list of expenses` part>

2. The user requests to mark or unmark the selected expense.


Also can add an extension that the user filters the list before requesting to mark or unmark the selected expense

Reply:
@rollingpencil For the "The system shows a list of expenses" part, I actually find this quite okay. Because if u take a look at our existing use cases section, it showcases the interaction of both **"the system"** and **"the user"**. As long as we have defined what **"the system"** is _(which we did)_, then is fine.

Reply:
@GavinSin You are missing the point. I think @rollingpencil point is that we should add the underline reference. No worries i will take over this PR from now on as I reevaluated our use cases and there seems to be a lot of changes required.